Strategies/TransitionToServices.md

```diff
@@ -0,0 +1,136 @@
+# A Strategy to transition IITB to a services oriented organization
+
+This document articulates a strategy and roadmap for moving IITB into a service provider by transforming IITB's Solution Delivery capability.
+
+The primary strategic goal of this transformation is to enable more rapid delivery of business-focused services at ESDC.
+
+This goal will be achieved through the establishment and execution of the following **7 key strategic initiatives**:
+
+1. Create an Enterprise Information Model
+2. Accelerate Cloud Adoption
+3. Invest in an Integration Layer (API-first approach)
+4. Implement DevOps
+5. Shift to Product Management
+6. Adopt a Bi-Modal Governance and Delivery
+7. Truly work in the open
+
+## 1. Create an Enterprise Information Modeling
+
+The creation of a stable, and highly re-usable set of APIs that expose relevant data and functionality is dependent upon the creation and continuous evolution of an Enterprise Information Model at ESDC. Such model will also enable ESDC to move towards a "tell us once" goal.
+
+The Enterprise Information Model is *not* a database schema, a particular system's logical data model, or a client profile. It's a reference, with business terms, that development teams will use to quickly identify:
+
+- data entities being created and managed at ESDC
+- the inter-relationships between those entities
+- their source of truth (system of record, data owner(s), data stewards, and API endpoints)
+
+Any privacy concerns will be treated separately, outside of the Enterprise Information Model.
+
+The Enterprise Information Model is part of a broader undertaking by IITB's Information Architecture directorate that seeks to mature ESDC's overall Information Architecture capability.
+
+## 2. Accelerate Cloud Adoption
+
+IITB will aggressively seek opportunities to adopt cloud computing and reap the benefits of rapid elasticity and increased agility.
+
+Priority will be given to SaaS and PaaS service models in a Public Cloud provider.
```

Comment:
Not sure what this is intended to mean ... priority over what? 

Could we summarive this in one sentence. Cloud first. 

?

Reply:
I would actually move towards a Cloud Native organisation. That's what the industry calls it and it makes way more sense:

- Leverage Cloud services first
- If building on premise, you can build in a way that natively supports Cloud even though you may not already be able to do so yet.

Reply:
Removed the line in latest version of this document.